auxiliaries.py

```diff
@@ -211,43 +212,21 @@ def __init__(self,
                  pinging=True,
```

Comment:
remove `up_interval_s`, `down_interval_s`, `pinging` arguments since they shouldn't be needed anymore